tools/release/publish_release.sh

```diff
@@ -604,30 +591,25 @@ fi
 if [ -z "$start_version" ]; then
     if [[ "$1" == "" ]]; then
         # grab latest draft excluding test version 9.99.9
-        draft=`gh release list | $GREP_CMD Draft | $GREP_CMD -v 9.99.9`
+        draft=$(gh release list | $GREP_CMD Draft | $GREP_CMD -v 9.99.9)
         if [[ "$draft" != "" ]]; then
-            target_version=`echo $draft | awk '{print $1}' | cut -d '-' -f2`
-            start_version=`gh release list | $GREP_CMD Draft -A1 | tail -n1 | awk '{print $1}' | cut -d '-' -f2`
+            target_version=$(echo $draft | awk '{print $1}' | cut -d '-' -f2)
+            start_version=$(gh release list | $GREP_CMD Draft -A1 | tail -n1 | awk '{print $1}' | cut -d '-' -f2)
         else
-            start_version=`gh release list | $GREP_CMD Latest | awk '{print $1}' | cut -d '-' -f2`
+            start_version=$(gh release list | $GREP_CMD Latest | awk '{print $1}' | cut -d '-' -f2)
         fi
     else
         start_version="$1"
     fi
 fi
 
-if [[ "$main_release" == "true" ]]; then
-    # Main releases are always minor releases
-    minor=true
-fi
-
 if [[ $start_version != v* ]]; then
-    start_version=`echo "v$start_version"`
+    start_version=$(echo "v$start_version")
 fi
 
 if [[ "$target_version" != "" ]]; then
     if [[ $target_version != v* ]]; then
-        target_version=`echo "v$target_version"`
+        target_version=$(echo "v$target_version")
```

Comment:
were these lint's you replaced? In some instances I had to change them from $() to `` for subtle bash reasons

Reply:
Yes, I use ShellCheck and it was yelling at me. The reason is [SC2006](https://www.shellcheck.net/wiki/SC2006). Does this break anything? I'm not real opinionated about this but that page lists some good reasons to `$(...)` notation instead of backticks.